server/start.go

```diff
@@ -89,14 +89,17 @@ func startStandAlone(ctx *Context, appCreator AppCreator) error {
 		cmn.Exit(err.Error())
 	}
 
-	// wait forever
 	cmn.TrapSignal(ctx.Logger, func() {
 		// cleanup
 		err = svr.Stop()
 		if err != nil {
 			cmn.Exit(err.Error())
 		}
 	})
+
+	// run forever (the node will not be returned)
+	select {}
+
 	return nil
```

Comment:
I'm getting this error when I run make with this commit:

/Users/tnachen/go/bin/golangci-lint run
server/start.go:103:2: unreachable: unreachable code (govet)
	return nil

Reply:
You're right, apparently for some reason this snippet dodged Golangci bot check: https://golangci.com/r/github.com/cosmos/cosmos-sdk/pulls/4884